go.mod

```diff
@@ -76,3 +71,10 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.1 // indirect
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
+
```

Comment:
These seem to be controller code dependencies. If we only add CRD type without CRD controller code, I don't think we need to update `go.mod` and `go.sum`.